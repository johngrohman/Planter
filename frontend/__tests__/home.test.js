import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../src/app/(tabs)';

test('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
});