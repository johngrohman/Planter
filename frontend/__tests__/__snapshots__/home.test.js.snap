// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text>
  Home page
</Text>
`;
